stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/RetryContext.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
```

Comment:
Some of the new files are with 2019 version and some are in 2020. Shall we make them all 2020?